src/manifest.json

```diff
@@ -13,7 +18,8 @@
     },
     "content_security_policy": "script-src 'self' 'unsafe-eval' https://az416426.vo.msecnd.net; object-src 'self'",
     "background": {
-        "page": "background/background.html"
+        "page": "background/background.html",
+        "persistent": false
```

Comment:
This seems to be a sensible change. Do we need to fully test our extension? I've been looking at the auto update feature and this came up... wondering how "dangerous" is to make this change, in the sense of the unintended consequences it may have...

Reply:
The EdgeHTML browser [requires](https://docs.microsoft.com/en-us/microsoft-edge/extensions/troubleshooting) a value be set here. The conservative approach would be to specify the default value to preserve the existing state. Google is so insistent on wanting you to set `"persistent": false` that they don't really [document](https://developer.chrome.com/extensions/manifest) what the default is. [This random blog post](http://coryg89.github.io/technical/2013/08/13/how-to-create-your-own-chrome-extensions/) from 2013 suggests that the default is `"persistent": true`. We do want to get to `"persistent": false` eventually, but I agree with @smoralesd that we'd need to test thoroughly before making this change.

Reply:
I'll do some testing as part of my spike... I'm fine with having this change as part of this PR I just want us all to be aware that maybe unintended consequences :(

Reply:
@peterdur I was under the assumption it was false since they seem very insistent on telling us to set it to false 😆 . I've changed it to true now.